api/v1/linstorsatellite_types.go

```diff
@@ -46,6 +46,15 @@ type LinstorSatelliteSpec struct {
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	Properties []LinstorNodeProperty `json:"properties,omitempty"`
+
+	// InternalTLS configures secure communication for the LINSTOR Satellite.
+	//
+	// If set, the control traffic between LINSTOR Controller and Satellite will be encrypted using mTLS.
+	// The Controller will use the client key from `LinstorCluster.spec.internalTLS` when connecting.
+	// +kubebuilder:validation:Optional
+	// + Without "nullable" the k8s API does not accept patches with 'internalTLS: {}', which seems to be a bug.
```

Comment:
Is the `+` necessary?

```suggestion
	// Without "nullable" the k8s API does not accept patches with 'internalTLS: {}', which seems to be a bug.
```

Reply:
Yes, this hides the content from the auto-generated docs of the CRDs, where I think this comment isn't really useful.